### PR TITLE
Disable ESLint Rule & Clean Up Example Env File

### PR DESCRIPTION
**Pull request by ipbyrne**
This PR disables the ESLint rule that was causing `docker-compose build` to fail. It also removes an unused variable from the example ENV and links up all of the example env values in the docker-compose.yml file so they will all get passed through to docker.

### CODE REVIEW

File: packages/dashboard/.env.example [hunk 1]
```diff
@@ -17,6 +16,8 @@ PORT=3000
 
 # Required if using 'elem:ganache'
 ELEMENT_CONTENT_ADDRESSABLE_STORE_SERVICE_URI='/ip4/127.0.0.1/tcp/5001' 
+ELEMENT_ANCHOR_CONTRACT=''
 ETHEREUM_RCP_URL='http://localhost:8545'
 ETHEREUM_PROVIDER=$ETHEREUM_RCP_URL
-MNEMONIC="your mnemonic phrase here"
\ No newline at end of file
+MNEMONIC="your mnemonic phrase here"
```
Comment (acarnagey) [inline packages/dashboard/.env.example:22]: I don't think `MNEMONIC` is being used, we could probably take out, same with docker-compose ref: /packages/dashboard/config/index.ts

**Review by acarnagey — APPROVED** [review]

File: packages/dashboard/.env.example (hunk 1 above)
Comment (OR13) [reply packages/dashboard/.env.example:22]: how are you able to "sign ropsten transactions"... without providing one?

File: packages/dashboard/docker-compose.yml [hunk 2]
```diff
@@ -6,20 +6,21 @@ services:
...
+      - DATABASE_NAME=${DATABASE_NAME}
+      - MAX_CONCURRENT_DOWNLOADS=${MAX_CONCURRENT_DOWNLOADS}
+      - BATCH_INTERVAL_IN_SECONDS=${BATCH_INTERVAL_IN_SECONDS}
+      - OBSERVING_INTERVAL_IN_SECONDS=${OBSERVING_INTERVAL_IN_SECONDS}
+      - ELEMENT_CONTENT_ADDRESSABLE_STORE_SERVICE_URI=${ELEMENT_CONTENT_ADDRESSABLE_STORE_SERVICE_URI}
+      - ELEMENT_ANCHOR_CONTRACT=${ELEMENT_ANCHOR_CONTRACT}
+      - ETHEREUM_RPC_URL=${ETHEREUM_RPC_URL}
+      - ETHEREUM_PROVIDER=${ETHEREUM_PROVIDER}
+      - ETHEREUM_MNEMONIC=${ETHEREUM_MNEMONIC}
+      - MNEMONIC=${MNEMONIC}
```
Comment (OR13) [inline packages/dashboard/docker-compose.yml:23]: mnemonic is here.

File: packages/dashboard/.env.example (hunk 1 above)
Comment (ipbyrne) [reply packages/dashboard/.env.example:22]: Copy that! Will pull it out of example and docker now!

Comment (OR13) [reply packages/dashboard/.env.example:22]: imo, the example config should be the same config that we use to for the test method... with real values as much as possible.

**Review by OR13 — APPROVED** [review]

File: packages/dashboard/.env.example (hunk 1 above)
Comment (acarnagey) [reply packages/dashboard/.env.example]: using `ETHEREUM_MNEMONIC` instead https://github.com/transmute-industries/sidetree.js/blob/main/packages/dashboard/config/index.ts#L18

**Review by BenjaminMoe — APPROVED** [review]

**Pull request merged**